CONTRIBUTING.md

```diff
@@ -6,18 +6,25 @@ Every little help and credit will always be given.
 There are multiple ways to contribute to Taipy, code, but also reporting bugs, creating feature requests, helping
 other users in our forums, Stack Overflow, etc.
 
-Today the only way to communicate with the Taipy team is by GitHub issues.
+Several channels are open to communicate with the Taipy R&D team:
 
-## Never contributed on an open source project before ?
+- Communicating through the [taipy.io forum](https://www.taipy.io/community/)
+- Through [GitHub issues](https://github.com/Avaiga/taipy-doc/issues)
+- etc.
```

Comment:
I would specify the goal of each channel, and I would add the StackOverflow channel.

Reply:
I don't think I would be able to do these specifications precisely as I'm new to the project. Could you provide me the infos for that part, please ?

Reply:
Sure.

I would recommend using: 
- The forum for any open discussions with the Taipy team.
- A new GitHub issue on the taipy repository to propose an improvement, a new feature, or report a bug.
- A new entry in the GitHub discussion page on the taipy repository to open a discussion on a technical topic related to the implementation of Taipy.
- StackOverflow for precise questions related to the usage of Taipy.

What do you guys think?

Reply:
I don't know for @florian-vuillemot or @FabienLelaquais but it seems fine to me. Do you have a general Taipy topic URL on Stackoverflow ?

Reply:
@jrobinAV I updated the section with your suggestions.